lib/chat_web/live/main_live/layout/card.ex

```diff
@@ -0,0 +1,61 @@
+defmodule ChatWeb.MainLive.Layout.Card do
+  @moduledoc false
+  use ChatWeb, :component
+
+  alias Chat.Card
+  alias Chat.Rooms.Room
+  alias Chat.Utils
+
+  @basic_text_style "text-sm"
+  @grayscale_text_style "text-sm tracking-tighter text-grayscale600"
+  @white_hash_text_style "text-base text-white/60"
+  @white_name_text_style "text-base font-bolt text-white t-peer-name"
+  @purple_text_style "font-bold text-sm text-purple"
+
+  attr :card, Card, doc: "room/user card"
+  attr :room, Room, doc: "room sctruct"
+  attr :me, Identity, doc: "current user"
+  attr :is_me?, :boolean, doc: "is this the current user's card?"
+  attr :stylized_as, :atom, default: :chat_list
+
+  def details(assigns) do
```

Comment:
rename to `hashed_name`
this way markup will have a clue of what is going on.

btw @doc is welcome